bfillings/swarm_v127.py

```diff
@@ -74,7 +74,9 @@ def __call__(self, seq_path):
             self._apply_identical_sequences_prefilter(seq_path)
 
         # Run Swarm
-        super(Swarm, self).__call__(seq_path)
+        result = super(Swarm, self).__call__(seq_path)
+        if result['ExitStatus'] != 0:
```

Comment:
@antgonza, this should instead be done by adding an ``_accept_exit_status`` method (see [here](https://github.com/biocore/burrito/blob/master/burrito/util.py#L470) for the base class method). All this would need to be is:

```
def _accept_exit_status(self):
    return result['ExitStatus'] == 0
```

This will automatically be called in the base class's ``__call__``.